src/adapter.js

```diff
@@ -331,12 +389,15 @@ class MongooseAdapter {
     if (fieldIndex <= 5 && fieldIndex + fieldValues.length > 5 && fieldValues[5 - fieldIndex]) {
       where.v5 = fieldValues[5 - fieldIndex];
     }
-
-    await CasbinRule.deleteMany(where);
+    const options = {};
+    if (this.useTransaction) options.session = await this.getTransaction();
```

Comment:
It is a matter of taste, but I don't like one-line if-s. Can you integrate `eslint` here with [standard rules](https://standardjs.com)?

Reply:
Maybe it's finally time to check why my VSCode does not like to import some of the eslint configurations. I'll update these to standard when I figure that one out.

Reply:
In general though, standard does not forbid one-line if-s. I think they are good if the code remains readable and understandable.

Do you feel that these requirements are not met?

Reply:
IDK, honestly. I get used to if statements with block in any case and these one-liners a little unusual for me. It is not a requirement, as you thought, relax 😄 

But seriously, it will be great to have at least `eslint --fix` for style issues

Reply:
Code styling should be fixed in the latest commit. It was mainly just removal of semicolons so far.

In general I just want it to clear what the code does and it's easy to read. Whatever style it is in the end, I do not care. I have just gotten use to one-liners and it makes code less "deep" or "nested".  But that's not valuable if it undermines readability or understandability. :)